spec/requests/api/v1/items_request_spec.rb

```diff
@@ -83,4 +83,68 @@
     expect(item[:data][:attributes][:description]).to be_a(String)
     expect(item[:data][:attributes][:unit_price]).to be_a(Float)
   end
+
+  it "creates an item" do
+    merchant = FactoryBot.create(:merchant)
+    item_params = {
+                  "name": "value1",
+                  "description": "value2",
+                  "unit_price": 100.99,
+                  "merchant_id": merchant.id # couldn't hardcode the number due to thousands of merchants being created through running tests
+                  }
+
+    headers = { "CONTENT_TYPE" => "application/json" }
+
+    post "/api/v1/items", headers: headers, params: JSON.generate(item: item_params)
```

Comment:
Using headers here works but so does simply wrapping the params in curly brackets, so if we were to replace lines 96-98 with simply: `post "/api/v1/items", params: {item: item_params}` I believe it would still work

Reply:
I stand corrected! Thanks for walking me through this on our huddle :)